dashboard/test/models/census/census_summary_test.rb

```diff
@@ -142,6 +142,22 @@ class Census::CensusSummaryTest < ActiveSupport::TestCase
     assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: nil, is_k8_school: nil)
   end
 
+  test "School with different AP school codes in different years functions correctly" do
```

Comment:
This is what I was thinking of!  Test ability to create two ap school codes for the same school for different years with different codes.  I think these tests belong in `ap_school_code_test.rb` though.

Reply:
I was thinking of putting it in `census_school_test.rb`, but that doesn't exist -- what assertion would you suggest using then (instead of checking the summarization)? FWIW, the `validate_summary` approach I used here actually caught a problem.